src/BIMDataComponents/BIMDataInput/BIMDataInput.vue

```diff
@@ -6,7 +6,7 @@
       success,
       disabled,
       loading,
-      'not-empty': modelValue != null,
+      'not-empty': modelValue !== '' || !!modelValue,
```

Comment:
`!!modelValue` returns the same than `modelValue !== ''` is `modelValue` equals `''`. Maybe `'not-empty': !!modelValue` is enough?